src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/Current_Routine_DeleteOrphanedLocations.sql

```diff
@@ -0,0 +1,12 @@
+﻿/**
+  Script which deletes Locations not referenced by any Observations.
+  Orphaned Locations occur when all Observations referencing them are deleted.
+ */
+CREATE OR
```

Comment:
Do we need a `Current_Routine_UpdateLocationGeographicLevel.sql` file as well?

Reply:
Thanks, I didn't bother creating this because I plan to drop it as soon as the next part is deployed which depends on GeographicLevel existing and being populated so I think it will be fairly short lived.

Reply:
Cool, I figured as much!